package.json

```diff
@@ -12,6 +12,7 @@
     "lint": "lerna run lint --stream",
     "build": "lerna run build --stream",
     "format": "lerna run format --stream",
+    "lint": "lerna run lint --stream",
```

Comment:
This can be removed i think as it is duplicated above from the merge of the other lint PR

Reply:
Thanks! Removed.